packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -229,7 +261,9 @@ export function publish(options: InitOptions): void {
       intercept(event); // making sure we can track changes to the Event object
       setTriggerFlowlet(event, flowlet);
       channel.emit('al_ui_event_capture', eventData);
-    },
+    };
+    window.document.addEventListener(eventName,
```

Comment:
better to call the `enableEventHandlers` to make sure we always use exact same mechanism.